ui-manchette/src/types.ts

```diff
@@ -51,7 +51,12 @@ export type OperationalPointType = ArrayElement<PathProperties['operational_poin
 export type StyledOperationalPointType = OperationalPointType & {
   styles?: CSSProperties;
   display?: boolean;
-  onClick?: (opId: string, opRef: HTMLDivElement | null) => void;
+  onClick?: (waypointIndex: number) => void;
+};
+
+export type WaypointMenuData = {
+  menu: React.ReactNode;
+  activeWaypointIndex?: number;
```

Comment:
Once #651 is merged, we'll be able to switch back to a waypoint ID here if we want to (and in the `onClick` callback)

Reply:
Ok so do I leave it like that and it will be dealt with in #651 or do we want to wait for it to be done ?

Reply:
I suppose it depends in which order the PRs get merged…